DockerHealthcheck.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if wget --spider -S "127.0.0.1:8080/login" 2>&1 | grep -w "200" ; then
```

Comment:
`wget` prints out all the headers and some other info. I think checking for just "200" word is not specific enough ...

Reply:
alright making it `HTTP/1.1 200 OK`

Reply:
Now it also checks for the `HTTP/1.1 200 OK` to be in the 2nd line.
It should always be at that line.